yagi/src/main/java/org/raml/yagi/framework/nodes/AbstractRamlNode.java

```diff
@@ -15,6 +15,8 @@
  */
 package org.raml.yagi.framework.nodes;
 
+import org.raml.yagi.framework.nodes.snakeyaml.SYIncludeNode;
```

Comment:
this class cannot depend on classes of the `snakeyaml` package, maybe we should add an `IncludeNode` interface that is implemented by `SYIncludeNode`